ui/src/components/view/InfoCard.vue

```diff
@@ -49,6 +49,14 @@
               </div>
             </slot>
           </div>
+          <div>
+            <br/>
+            <a-divider/>
+            <p><b>2FA QR</b></p>
+            <span v-if="resource.username">
+              <img src="https://www.google.com/chart?chs=200x200&chld=M%%7C0&cht=qr&chl=otpauth://totp/Apache%20CloudStack%3Atest%40gmail.com?secret=7t4gabg72liipmq7n43lt3cw66fel4iz&issuer=Apache%20CloudStack"/>
```

Comment:
TODO: This is harcoded, needs to be generated by backend API and shown only one time to the user (with maybe some backup codes)

Reply:
@rhtyd  nice work !

I have a question, if user has not logged in yet, how to get this QR code ?

Reply:
The workflow would be for the user to first log-in, then click a button to enable 2FA where they'll be shown this QR code. For this hack, I've taken shortcuts (for ex. the static QR code, instead of a randomly generated one per account).